data/schema.graphql

```diff
@@ -2615,6 +2615,9 @@ type ArtworkVersion implements Node {
   # The Artwork Version attribution class
   attributionClass: AttributionClass
 
+  # Artwork condition description
+  condition_description: String
+
```

Comment:
Why do we have these schema changes?

Reply:
I just run `yarn sync-schema` and it pulled these changes too